package.json

```diff
@@ -43,6 +43,7 @@
     "eslint-loader": "^2.1.1",
     "favicons-webpack-plugin": "0.0.9",
     "file-loader": "^2.0.0",
+    "fs-extra": "^7.0.1",
```

Comment:
don't forget to run `rm -rf node_modules && yarn install` to update _yarn.lock_ too.

Reply:
Good point